book/02-git-basics/sections/tagging.asc

```diff
@@ -238,9 +246,18 @@ To /git@github.com:schacon/simplegit.git
  - [deleted]         v1.4-lw
 ----
 
+The way to interpret the above is to read it as the null value before the colon is being pushed to the remote tag name, effectively deleting it.
+
+The second (and more intuitive) way to delete a remote tag is with:
+
+[source,console]
+----
+$ git push origin --delete <tagname>
+----
+
 ==== Checking out Tags
 
-If you want to view the versions of files a tag is pointing to, you can do a git checkout, though this puts your repository in ``detached HEAD'' state, which has some ill side effects:
+If you want to view the versions of files a tag is pointing to, you can do a `git checkout`, which puts your repository in ``detached HEAD'' state, which has some ill side effects:
```

Comment:
🤔 I think there are too many `which`s here. If you don't like the "though this" structure, maybe this should be two separate sentences.

Reply:
OK, mostly reverted that change.